VideoStreaming/VideoPlaylistViewController.swift

```diff
@@ -58,18 +57,48 @@ extension VideoPlaylistViewController: UITableViewDelegate, UITableViewDataSourc
     }
     
     func tableView(_ tableView: UITableView, numberOfRowsInSection section: Int) -> Int {
-        return playlist.count
+        return streams.count
     }
     
     func tableView(_ tableView: UITableView, cellForRowAt indexPath: IndexPath) -> UITableViewCell {
-        guard let cell = tableView.dequeueReusableCell(withIdentifier: VideoTableViewCell.Constants.reuseIdentifier, for: indexPath) as? VideoTableViewCell else {
+        
+        guard let cell = tableView.dequeueReusableCell(withIdentifier: VideoTableViewCell.Constants.reuseIdentifier,
+                                                       for: indexPath) as? VideoTableViewCell else {
             return UITableViewCell()
         }
-        guard let video = playlist[indexPath.row] else {
+        
+        guard let video = streams[indexPath.row] else {
             return UITableViewCell()
         }
+        
         cell.configure(with: video)
         return cell
     }
     
+    func tableView(_ tableView: UITableView, didSelectRowAt indexPath: IndexPath) {
+        let streamPlayerViewController = StreamPlayerViewController()
+        
+        if let streamId = streams[indexPath.row]?.id,
+            let path = Bundle.main.path(forResource: "video", ofType: "m3u8") {
```

Comment:
Me perdí aquí, este m3u8 es un archivo con los links de las URLs. El video player va a entender es una url de streaming supongo. No creo que soporte ese tipo de archivo, o me equivoco?

Reply:
No, lo que hice acá fue entrar al enlace que aparece en el ticket de 'API for the Streaming URL' y descargar un video de twitch para probar si el player lo reproducía. Ahora que lo comentas, si lo único que hay que hacer tener los enlaces que me trae cuando le mande la url del streaming encodeada, ahora creo una nueva branch y pruebo lo de ese ticket